apps/generator/docs/nunjucka-depreciate.md

```diff
@@ -0,0 +1,120 @@
+--
+title: "Depreciation of nunjucks render engine"
+weight: 170
+---
+
+# Migration Guide from nunjucks render engine to react render engine
```

Comment:
```suggestion
# Migration guide from Nunjucks render engine to React render engine
```

Reply:
do we need h1 here, and in the other cli deprecation PR?

in website title is rendered as h1 header right? so with such manual h1 header it will look kinda duplicated, no?